cmd/ipfs-cluster-service/main.go

```diff
@@ -189,10 +189,10 @@ func main() {
 			Name:  "debug, d",
 			Usage: "enable full debug logging (very verbose)",
 		},
-		cli.StringFlag{
-			Name:  "loglevel, l",
-			Value: defaultLogLevel,
-			Usage: "set the loglevel for cluster components only [critical, error, warning, info, debug]",
+		cli.StringSliceFlag{
+			Name:   "loglevel, l",
+			EnvVar: "LOG_LEVEL",
+			Usage:  "set loglevels for all or separate loglevels for individual log identifiers [loglevel, identifier:loglevel]. Valid loglevels are critical, error, warning, notice, info and debug.",
```

Comment:
Add a paragraph and/or an example to the description explaining how to set log levels, keep this line short.